maintenance/maintenance.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+
+# Halt on errors/unsets, change fail returns, change field separator
+#
+set -euo pipefail
+IFS=$'\n\t'
+
+
+# Parameters and mode variables
+#
+PROJECT=jcxjin-${1:-}
+COMMAND=${2:-}
+VERBOSE=${VERBOSE:-}
+
+
+# App and build settings
+#
+APPLICATION_NAME=${APPLICATION_NAME:-pims-app}
+ENVIRONMENT_NAME=${1:-}
+APPLICATION_PORT=${APPLICATION_PORT:-8080-tcp}
+STATIC_PAGE_NAME=${STATIC_PAGE_NAME:-proxy-caddy}
+STATIC_PAGE_PORT=${STATIC_PAGE_PORT:-2015-tcp}
+#
+IMG_SRC=${IMG_SRC:-bcgov-s2i-caddy}
+GIT_REPO=${GIT_REPO:-https://github.com/bcgov/pims.git}
+GIT_BRANCH=${GIT_BRANCH:-dev}
+OC_BUILD=${OC_BUILD:-../openshift/templates/maintenance/caddy.bc.json}
+OC_DEPLOY=${OC_DEPLOY:-../openshift/templates/maintenance/caddy.dc.json}
```

Comment:
please note that you added the templates in YAML format (which I prefer too as they are more readable). However, that would cause the script to fail as there is no JSON file in sight. Please rename to yaml here? Thanks